<h1>BIOMOD @CGU</h1>

<h1>Abstract</h1>
<h2>Biomolecule-based nanostructures are of great interests in bio-nanomaterial design due to their rich structural diversity and inherent biocompatibility. Among them, cyclic peptides have received remarkable attention owing to their rigid configuration as well as chemical and enzymatic stability. To date, previous reports have mainly focused on the construction of nanotubes in a ring-stacking manner through hydrogen bonding. However, other assembling strategies have rarely been explored. In this work, two cyclic peptides, cyclo-(DP)3 and cyclo-(DG)3, have been rationally designed and used as building blocks to fabricate novel assemblies. Nanovesicles and nanotubes were successfully constructed by crosslinking cyclo-(DP)3 with hexamethylendiamine or cyclo-RR based on covalent bonding or electrostatic interaction. To further explore its potential application in nucleotide delivery, plasmid DNA and oligoRNA were successfully bound to the vesicles and the latter was taken into the cell. On the basis of this result, more complex functions, such as gene delivery and targeted siRNA delivery, are expected to achieve with these cyclic peptide assemblies.</h2>

<h1 id="video" >Video</h1>
  <div><iframe width="800" height="504" src="https://www.youtube.com/embed/ExT4F3fYdIA&index=1&list=RDExT4F3fYdIA" frameborder="0" allowfullscreen><iframe</div>

<h1>Project</h1>

<ul>
  <h2><li>Background</li></h2>
 <p>Building really small things is typically extremely hard; however since the development of the DNA origami technique a new world of opportunity has been opened. DNA origami allows scientists to relatively easily build arbitrary shapes and 3-dimensional structures on the nanoscale – some examples of these are shown below.For more information of the DNA origami technique see DNA design. There are many applications for this technology; however one of the most interesting opportunities which emerged early on was the prospect of developing nanoscale vessels which could deliver molecular cargo. The series of events unfolds as follows: a vessel is loaded with the cargo, the vessel is delivered, and then a release mechanism unloads the cargo.DNA origami has been used to not only just capture and release cargo, but to autonomously target selective cells and release the contents directly to the target site via logic-gated mechanisms. </p>
  <h2><li>The problem</li></h2>
  <p>The process of targeted delivery can be broken down into three steps: loading the vessel with the drug of interest, recognising the target site, and releasing the payload directly at that site. As shown above with Douglas et al. and Ranjbar et al., effective vessels have already been developed, but thus far the method of loading the cargo is limited to chemically binding it to the vessel.</p>

<p>What if we could load a vessel with an unbound payload? So that when delivered rather than remaining attached to the vessel the cargo is free to diffuse out and perform its function. Would the free cargo be more effective compared to a bound cargo? Could it fit into smaller spaces and perform unique tasks that were previously unachievable due to being bound to the vessel?</p>

  <p>What if we could take it a step further and build a vessel which was selectively responsive to autonomously loading a particular cargo? Could that lay the foundation for a new generation of smart drug delivery vessels which can not only selectively unload their cargo, but can also identify free cargo in solution and load it?</p>
  <h2><li>Our solution</li></h2>
  <p>These are interesting questions to ask, but for the sake of practicality they should be considered one at a time. First, we needed to ask if it were possible to capture any molecular cargo unbound inside a vessel.</p>

<p>Capturing unbound cargo isn’t a trivial task. Resorting to probability and locking a vessel at an arbitrary time and hoping to capture a particle is a poor strategy due to the enormous dispersity of particles in solution relative to the confined cavity space of most vessels.</p>

<p>The only alternative, we found, was to design a vessel which could actually be triggered to close by the proximity of the target molecule. After hundreds of bad drawings and a few late nights on caDNAno, out popped our design (which ended up looking a lot like a mousetrap.)</p>

  <p>This is a good solution because it proposes a way to potentially capture a specific unbound molecular cargo inside of a DNA origami vessel, which due to the nature of DNA origami structures could be easily modified with receptors to mediate targeting to diseased cells and deliver a payload.</p>

<p>Even with the application of targeted delivery aside, capture of free molecules inside of molecular cages has proved to be a significant challenge and this project offers a novel solution. This design also creates opportunity for research and discovery regarding the mechanical properties of spring-loaded DNA origrami structures, such as torsional and longitudinal stiffness.</p>
  <h2><li>Project goal</li></h2>
<p>here is a trade-off between choosing goals that are realistic in the short timeframe of BIOMOD and which are also scientifically significant and exciting. The prospect of developing new DNA origami vessels which can selectively capture unbound molecular cargo is one which can be explored in many directions, but for this competition we have chosen strategic goals which allow us to test the core concept and its feasibility</p>
</ul>
   
<h1>Design</h1>

<ul>
<h2><li>DNA probe</li></h2>
  <p>The purpose of our project in the BIOMOD competition isn’t to capture a payload of any medicinal value, but rather to prove the concept that a payload can be captured unbound inside of a DNA origami box which could potentially be modified to facilitate targeted delivery.</p>
<p>There a number of potential directions to take this project from here on. Some interesting avenues and opportunities include:</p>
  <p>Testing different brace-releasing mechanisms other than via endonuclease</p>
<p>Cutting the DNA brace with a restriction enzyme is only one way of releasing the pressure and capturing a payload. For example perhaps a binding event could take place in which the DNA in the brace binds to an accessory on the payload and thus stops supporting the lids causing the box to collapse.</p>

<p>This is worthwhile exploring as it may offer a more elegant solution with less or perhaps no modification to the payload of interest in order to capture it. Different variations of brace structures and receptors could exploit natural properties of the payload of interest to trigger selective closing.</p>

<h2><li>Chip</li></h2>
<p>Potential cargo release mechanisms</p>
<p>The next step is obviously to modify or design a new spring-loaded structure which can not only trap but also release the molecular payload. A couple of potential strategies include:</p>
<ul>
  <li>Star activity of the endonuclease could eat out of the origami at body temperature. Perhaps the origami could be ingested in a frozen tablet to strategically time the degradation.</li>
  <li>Addition of surface receptors for internalisation of origami into T-cells or otherwise where the harsh environment will degrade the origami and release the payload.</li>
  <li>Reformation of the brace at the target site.</li>
  <li>Destruction of the hinge region to separate the lids.</li>
</ul>
</ul>
   
<h1>Experiment</h1>

<ul>
<h2><li>Purification</li></h2>
  <p>All the assemblies synthesized using amination reaction were purified by three days of dialysis against DMF and another three days of dialysis against water. Then, the assemblies were enrichd using ultrafiltration membranes (100KD, Millipore). The assemblies constructed through electrostatic interaction were directly characterized without purification.</p>
<h2><li>Synthesis of Cyclic Peptides</li></h2>
  <p>The linear peptide precursor was synthesized using solid phase peptide synthesis (SPPS) method. The cyclic peptides were synthesized either in a dilute DMF solution (0.1 mmol/L) or using ion-assisted method (5 mmol/L). The cyclic peptides used for self-assembly were purchased from the above-mentioned companies and directly used without further purification.</p>
<h2><li>Dynamic Light Scattering and Zeta Potential Measurements</li></h2>
  <p>The size and the zeta potential of the cyclo-peptide complexes were measured by dynamic light scattering using a Zetasizer NANO ZS90 (Malvern Instruments, UK). Three repeat measurements were made for each complex.</p>
<h2><li>Preparation of Assemblies/Plasmid DNA (pLVX-shRNA1) Complex</li></h2>
  <p>The assemblies/plasmid DNA (pLVX-shRNA1) complex was prepared as described below: 20 μL assemblies was mixed with 2 μL, 79 ng/μL solution of plasmid DNA (pLVX-shRNA1) ; All the plasmid DNA are generously donated by Dr. Haihong Zhang from Engineering Laboratory for AIDS Vaccine, Jilin University. The resulting complexes were kept for 24 hours at room temperature prior to use.</p>
<h2><li>Cell Culture and Transfection</li></h2>
  <p>Human breast cancer (MCF-7) cells were maintained in RPMI-1640 growth medium (Hyclone, South Logan, USA) supplemented with 10% fetal bovine serum (FBS, Gibco, Grand Island, USA), 1% v/v penicillin/ streptomycin solution (10000 U mL-1/10 mg mL-1) at 37℃ in an atmosphere of 5% CO2. MCF-7 cells were seeded onto 24-well plates (NEST, Wuxi, China) at a density of 5.0 × 105 cells per well and allowed to adhere overnight. The cells were serum starved and treated with 500μl of the oligo RNA, or assemblies/oligo RNA complexes. Following 6 hours of incubation, the plate was washed three times by PBS and the cells were incubated with fresh serum-containing medium for another 2 hours at 37℃ in an atmosphere of 5% CO2 before further analysis. Images of cells were obtained by fluorescence microscope (OLYMPUS IX71, Japan)</p>
</ul>

<h1>Result</h1>

<ol>
<h2><li>Synthesis and Characterization of the Assemblies</li></h2>
  <p>In this work, the assemblies were synthesized in the following two manners: (1) direct condensation of carboxyl group and amino group and (2) electrostatic interaction between carboxyl group and guanidyl group. After assembling, the morphologies of the assemblies were characterized by scanning electron microscopy (SEM). SEM images of the assemblies revealed that different combinations of peptide rings and linkers could assemble to form different morphologies. By covalently crosslinking cyclo-(DG)3 with ethylenediamine, vesicles with a diameter about 200 nm were successfully obtained (Figure 1-1 (a)). However, when cyclo-(DP)3 was crosslinked with ethylenediamine, nanosheets were fabricated instead of vesicles (Figure 1-1 (b) and (c)).</p>
  <p>The morphologies of the assemblies constructed with different combinations of rings and linkers were summarized in Table 1. These results showed that covalent crosslinking of rigid rings and short linkers would lead to the formation of nanosheets, while rigid rings with long linkers or soft rings with short linkers would be favorable for the formation of vesicles. Additionally, soft rings and long linkers would assemble to form amorphous aggregates. With the decreasing of rigidity, the morphology varied from sheet to vesicle, and finally became amorphous aggregates, which demonstrated that the assemblies’ morphology was highly dependent on the rigidity of formed nanostructure. The rigidity of the assemblies constructed in this fashion could be easily tuned by changing the amino acid components of peptide rings or adjusting the linker’s length. This method might also be useful for the construction of biomolecule-based membrane and even monolayer biomolecule-based nanosheets.</P>
<h2><li>Fusing Mechanism: a Possible Mechanism for the Generation of Large Vesicles and Tubes</li></h2>
<p>Several significant differences can be concluded from the SEM studies between the assemblies synthesized with cyclo-(DP)3 and 1, 6-diaminohexane (Assemblies 1) and the assemblies constructed with cyclo-(DP)3 and cyclo-RR (Assemblies 2). First of all, while assemblies 1 maintained their spherical morphology with the increasing ratio of carboxyl group to amino group, assemblies 2 are responsive to the ratio of carboxyl group to guanidyl group, switching their morphology between vesicles and tubes. Secondly, the sizes of the vesicles from assemblies 1 mostly range from 100 nm to 300 nm, but the sizes of the vesicles from assemblies 2 are all over 1 μm. Thirdly, while the vesicles from assemblies 1 have a strong tendency for aggregation, vesicles constructed through electrostatic interaction are mostly dispersed.</p>
<h2><li>Applications in Nucleic Acid Delivery</li></h2>
<p>As shown in figure 1-3 (a) and (b), the vesicles are mostly in the range of 100 nm to 300nm and positively charged after dialysis against water. Since particles in this range exhibit a significantly improved cellular uptake efficiency7 and their positive charge can be used to load nucleic acids, they are excellent candidates for nucleic acid delivery.</P>
<p>To test their ability to adhere nucleic acids, condensed assemblies 1 were mixed with plasmid DNA for 24 hours and oligoRNA for 9 hours. After this, agarose gel electrophoresis was used to detect whether plasmid DNA and oligoRNA could be adhered to the vesicles. As shown in figure 4-1(a), fluorescence completely disappeared when plasmid DNA were mixed with the vesicles, which implied that plasmid DNA have been attached to the vesicles. Figure 4-1(b) showed that after being mixed with assemblies 1, the fluorescence intensity decreased, which implied that oligoRNA could also be attached to the vesicles.</p>
<h2><li>Conclusion</li></h2>
<p>In this work, we have presented a novel strategy to construct nanoscale assemblies with cyclic peptide. The morphology of the assemblies varies under different conditions. While nanovesicles and nanosheets have been fabricated through amidation reaction by modulating the assemblies’ rigidity, nanovesicles and nanotubes can be constructed through electrostatic interaction with different ratios of linkers to rings. An easy approach has been proposed for size-controllable synthesis of nanovesicles based on the fusing mechanism, which was further proved by DLS studies. The vesicles synthesized using amidation reaction were found capable of adhering nucleic acid. Their potential application in nucleic acid delivery was revealed by fluorescence microscope image that FAM-tagged oligoRNA was successfully delivered to the cells.</p>
<p>On the basis of these results, we hope our boldly proposed methods can contribute to the diversity of nucleic acid delivery and give some inspiration to the field of controllable nano-synthesis. Theoretical calculation has revealed that pH-responsive delivery of nucleic acid can be achieved by incorporating histidine to the cyclic peptide. And due to their built-in structural flexibility, more complicated functions can be realized by carefully designing the cyclic peptides’ structure. In the future, assemblies constructed using this strategy may also be useful in the fields of separation, catalysis and energy, which are far beyond the usage in nucleic acid delivery.</p>
<h2><li>REFERENCES</li></h2>
<p>1.   Lovell, S. C.; Davis, I. W.; Adrendall, W. B.; de Bakker, P. I. W.; Word, J. M.; Prisant, M. G.; Richardson, J. S.; Richardson, D. C., Structure validation by C alpha geometry: phi,psi and C beta deviation. Proteins-Structure Function and Genetics 2003, 50 (3), 437-450.</p>
<p>2.   Kim, D.; Kim, E.; Lee, J.; Hong, S.; Sung, W.; Lim, N.; Park, C. G.; Kim, K., Direct Synthesis of Polymer Nanocapsules: Self-Assembly of Polymer Hollow Spheres through Irreversible Covalent Bond Formation. Journal of the American Chemical Society 2010, 132 (28), 9908-9919.</p>
<p>3.    Baek, K.; Yun, G.; Kim, Y.; Kim, D.; Hota, R.; Hwang, I.; Xu, D.; Ko, Y. H.; Gu, G. H.; Suh, J. H.; Park, C. G.; Sung, B. J.; Kim, K., Free-Standing, Single-Monomer-Thick Two-Dimensional Polymers through Covalent Self-Assembly in Solution. Journal of the American Chemical Society 2013, 135 (17), 6523-6528.</p>
<p>4.    Varkouhi, A. K.; Scholte, M.; Storm, G.; Haisma, H. J., Endosomal escape pathways for delivery of biologicals. Journal of Controlled Release 2011, 151 (3), 220-228.</p>
<p>5.    Vauthey, S.; Santoso, S.; Gong, H. Y.; Watson, N.; Zhang, S. G., Molecular self-assembly of surfactant-like peptides to form nanotubes and nanovesicles. Proceedings of the National Academy of Sciences of the United States of America 2002, 99 (8), 5355-5360.</p>
  </ol>
<h1>Future</h1>
  
<h1>Team</h1>
  
<h1>Sponsors</h1>
